src/components/Shared/FormShell.js

```diff
@@ -50,7 +52,9 @@ export default function FormShell({
   setIDInfoAutoFilled,
   visibleClear,
   actions,
-  filteredItems = []
+  filteredItems = [],
+  isClearedGrid = false,
+  isClearedAll = false
```

Comment:
not possible to add only clearGrid and for clearAll to use the current clear we had already?

Reply:
we need the confirmation message